doc/developer/diagnostic-queries.md

```diff
@@ -127,3 +127,47 @@ where
 group by mdo.id, mdo.name
 order by sum(mas.records) desc;
 ```
+
+### How can I check whether work is distributed equally across workers?
+
+Work is distributed across workers by the hash of their keys. Thus, work can
+become skewed if situations arise where Materialize needs to use arrangements
+with very few or no keys. Example situations include:
+* views that maintain order by/limit/offset
+* cross joins
+* joins where the join columns have very few unique values.
+
+Currently, only one worker ingests a partition, so there could be ingest skew if
+some partitions have more records than others.
```

Comment:
This text could probably be a bit more clear. This is probably about sources, using Kafka-only terminology. I'm not certain the intent, but if it is to flag another case that there could be skew that shouldn't be in the list above (presumably just about arrangements), then maybe

> Additionally, the operators that implement data sources may demonstrate skew, as they (currently) have a granularity determined by the source itself, for example the number of parts in a Kafka topic.